external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/trident/KafkaTridentSpoutEmitter.java

```diff
@@ -279,6 +293,18 @@ private long seek(TopicPartition tp, KafkaTridentSpoutBatchMetadata lastBatchMet
         return fetchOffset;
     }
 
+    /**
+     * Seek the consumer to offset corresponding to startTimeStamp.
+     */
+    private void seekOffsetByStartTimeStamp(TopicPartition tp) {
+        long startTimeStampOffset;
+        Map<TopicPartition, OffsetAndTimestamp> map = consumer.offsetsForTimes(Collections.singletonMap(tp, startTimeStamp));
```

Comment:
Nit: Not a very useful variable name. How about `offsetsForTimes`?

Reply:
Np